### ISSUE TITLE
[Arch] Distribution Support

### ISSUE THREAD

**Issue by TylerBrock** [nodkz/mongodb-memory-server#302]
When i set binary.version to '4.2.5' i get an error that it doesn't exist, not sure if that is an upstream issue or an issue with this library but 4.2.5 is definitely available.

**Comment by nodkz** [comment]
Upstream issue. According to your operation system, you may use only available versions from upstream
https://www.mongodb.org/dl/linux

What OS do you use?

**Comment by TylerBrock** [comment]
I'm on Arch Linux, is there a generic linux build that isn't tied to distribution?

**Comment by TylerBrock** [comment]
I looked at what the AUR package for mongodb-bin does and it looks like it extracts the sever/shell/etc... from the ubuntu bionic debs.

**Comment by nodkz** [comment]
Before we don't find any distribution of mongodb bins for Arch Linux. If you found working bins for Arch Linux, please provide a download link that will work for you or PR.

The code is quite simple:
https://github.com/nodkz/mongodb-memory-server/blob/master/packages/mongodb-memory-server-core/src/util/MongoBinaryDownloadUrl.ts
https://github.com/nodkz/mongodb-memory-server/blob/master/packages/mongodb-memory-server-core/src/util/__tests__/MongoBinaryDownloadUrl-test.ts

**Comment by johndcarmichael** [comment]
Do you have an example of how to instruct this lib to download the 4.2 binary?

@nodkz 
Following the links above i think i got this correct that a proc env should be set: https://github.com/nodkz/mongodb-memory-server/blob/master/packages/mongodb-memory-server-core/src/util/MongoBinaryDownloadUrl.ts#L39

```
    "preinstall": "MONGOMS_DOWNLOAD_URL=https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1604-4.2.8.tgz",
```
I can see the bin gets downloaded after an npm i, so setting the said proc env as a preinstall i thought would get the package to download this version, but i am still left with mongo 4.0.14.. 

Meaning i cannot use the $regexMatch. 
```
    MongoError: Unrecognized expression '$regexMatch'
```

Do you have an example on how to get the system to install the correct binary anywhere?

**Comment by hasezoey** [comment]
closing because support is included since `7.0.0-beta.16`
